doc/configuration_options.rst

```diff
@@ -270,6 +270,13 @@ The following options are valid in version ``2`` (beside the generic options):
   * ``default``: a boolean flag as described for *test_commits*.
   * ``force``: a boolean flag as described for *test_commits*.
 
+* ``tests_require_gpu``: a dictionary to indicate if software tests needs gpu
+  support to run correctly.
+  * ``default``: a boolean flag as described for *test_commits*.
+  * ``run_only_gpu_tests:``` a dictionary flag to indicate if only gpu tagged
+  tests will be run.
+    * ``default``: a boolean flag as described for *test_commits*.
```

Comment:
The indentation of the last two bullets seems off. The bullet containing `run_only_gpu_tests` needs an empty line before.

Reply:
I've change it in d9a43bd